lib/blocks.php

```diff
@@ -220,6 +220,11 @@ function gutenberg_wpautop_block_content( $content ) {
  */
 function gutenberg_wpautop_insert_post_data( $data ) {
 	if ( ! empty( $data['post_content'] ) && gutenberg_content_has_blocks( $data['post_content'] ) ) {
+		// WP_REST_Posts_Controller slashes post data before inserting/updating
+		// a post. This data gets unslashed by `wp_insert_post` right before
+		// saving to the DB. The PEG parser needs unslashed input in order to
+		// properly parse JSON attributes.
```

Comment:
Does this mean we need to "slash" data back after our custom behavior?

Reply:
Ah, yes, I left that line out. Amended. (Initially, not slashing back didn't break anything for me, but obviously any other piece consuming the hook should expect slashed data, as the signature for the hook states.)